base/abstractarray.jl

```diff
@@ -797,6 +797,8 @@ function copy(a::AbstractArray)
     copymutable(a)
 end
 
+copy(a::SubArray) = a.parent[a.indices...]
```

Comment:
why not putting it in `subarray.jl` if it's about `SubArrays`s?

Reply:
I was unsure where to put it, so I did a search for where `function copy(` appeared that would seem relevant. Perfectly happy to move to subarray.jl if desired.

Reply:
I think I agree with moving it.

Reply:
Moved to subarray.jl